test/localization/spot_check_about_has_strings.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * spot check that each language has values for the string id keys on About page
+ * that are contained in English (i.e. make sure strings will show up, not ids")
+ */
+var tap = require('tap');
+var languages = require('../../languages.json');
+window = {};
+require('../../intl/about.intl.js');
```

Comment:
There definitely should be integration tests, but so long as unit tests catch the same errors as will break the system in the end, I don't think you need to test everything against the output of the build.  You could break this test up into a few different things — the build creates the file, it assigns to `window.messages`, and the keys for each language match those for `en`.  So long as the tests pass for each of these, then you're in the same place testing-wise even if you don't test the whole build.